paasta_tools/api/client.py

```diff
@@ -83,6 +84,30 @@ def get_paasta_api_client(
     return paasta_tools.api.auth_decorator.AuthClientDecorator(c, cluster_name=cluster)
 
 
+def get_paasta_oapi_client(
+    cluster: str = None,
+    system_paasta_config: SystemPaastaConfig = None,
+    http_res: bool = False,
+) -> Any:
+    if not system_paasta_config:
+        system_paasta_config = load_system_paasta_config()
+
+    if not cluster:
+        cluster = system_paasta_config.get_cluster()
+
+    api_endpoints = system_paasta_config.get_api_endpoints()
+    if cluster not in api_endpoints:
+        log.error("Cluster %s not in paasta-api endpoints config", cluster)
+        return None
+
+    parsed = urlparse(api_endpoints[cluster])
+    server_variables = dict(scheme=parsed.scheme, host=parsed.netloc)
+
+    config = paastaapi.Configuration(server_variables=server_variables)
+    client = paastaapi.ApiClient(configuration=config)
```

Comment:
do we need to do something here to integrate the PaastaRequestsClient class below just like we have with the swagger client?

Reply:
as i understand we can provide/modify relevant `ssl_*` attributes in `paastaapi.ApiClient`, i didn't test that yet. which environment do we use https in? in api_endpoints i only see http.

Reply:
Yeah, we never rolled it out anywhere. In theory it's setup in mesosstage but may or may not work in terms of issuing the certs. The other "magic" was here https://github.com/Yelp/paasta/blob/master/paasta_tools/api/auth_decorator.py which makes the client re-issue the cert if it is expired. Anyway, I'm okay if the SSL stuff breaks with this migration, we can always make it work if and when we come back to rolling out the authn layer.